components/scream/src/share/io/output_manager.hpp

```diff
@@ -110,12 +110,12 @@ class OutputManager
   ekat::Comm                                   atm_comm;
   ekat::Comm                                   pio_comm; 
   ekat::ParameterList                          m_params;
-  std::shared_ptr<const FieldManager<Real>> m_device_field_manager;
+  std::shared_ptr<const FieldManager<Real>>    m_device_field_manager;
   std::shared_ptr<const GridsManager>          m_grids_manager;
 
   bool                                 param_set = false;
   bool                                 gm_set    = false;
-  bool                                 fm_set  = false;
+  bool                                 fm_set    = false;
```

Comment:
Minor: these bools are kind of redundant. Since gm and fm are pointers, one can simply check if the pointers are null. E.g., replace `if (gm_set)` with `if (m_grids_manager)`.

Reply:
To give more context, checking the pointer directly does not require one to make sure that the bool and the pointer are "in sync" (a minor bug that can creep in during refactors, or if another method is added).